tools/arxiv/scripts/run_test_pipeline.py

```
@@ -0,0 +1,170 @@
+        print(f"❌ Pipeline script not found: {pipeline_script}")
+        return False
+    
+    cmd = [
+        sys.executable,
+        str(pipeline_script),
+        "--config", str(config_path),
+        "--count", str(count),
+        "--arango-password", arango_password
+    ]
+    
+    print(f"Running command:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)